test/readme_test.js

```diff
@@ -16,11 +16,11 @@ content = content.replace(
 const blocks = matchBlocks("js", content);
 
 for (let block of blocks) {
-  test(`run ${block.split("\n")[0]}`, async t => {
+  test(`run ${block.split("\n")[0]}`, async (t) => {
     try {
       await exec(block);
     } catch (err) {
-      console.log(err);
+      t.log(err); // prints log alongside the test instead of immediately printing them to stdout
```

Comment:
ah nice. TIL

Reply:
My tests were failing and I couldn't debug them efficiently because the logs were confusing. Then I found this. 

Turns out adding a comment makes the readme test fail.
```js
getLogs(options, {
   fromBlock: '0xcb83ce' // Adding this comment cause the test to fail
})
```